ios/App/App.xcodeproj/project.template.pbxproj

```diff
@@ -359,6 +359,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = ${VERSION_CODE};
 				DEVELOPMENT_TEAM = CHLBG6683N;
+				GOOGLE_REVERSED_CLIENT_ID = "${GOOGLE_REVERSED_CLIENT_ID}";
```

Comment:
+1 - good to see this pattern working, will make it much easier to avoid need for hardcoded changes.

I'm also guessing if we need to opt out in the future we could just do a regex search for the templated string and remove the line when templating (as part of the env-parser script), but not a priority for now